pandas/core/series.py

```diff
@@ -1316,8 +1316,74 @@ def unique(self):
 
         return result
 
-    @Appender(base._shared_docs['drop_duplicates'] % _shared_doc_kwargs)
     def drop_duplicates(self, keep='first', inplace=False):
+        """
+        Return Series with duplicate values removed.
+
+        Parameters
+        ----------
+        keep : {'first', 'last', ``False``}, default 'first'
+            - 'first' : Drop duplicates except for the first occurrence.
+            - 'last' : Drop duplicates except for the last occurrence.
+            - ``False`` : Drop all duplicates.
+        inplace : boolean, default ``False``
+            If ``True``, performs operation inplace and returns None.
+
+        Returns
+        -------
+        deduplicated : Series
+
+        See Also
+        --------
+        Index.drop_duplicates : equivalent method on Index
+
```

Comment:
can also DataFrame.drop_duplicates. and Series.duplicated

Reply:
can you edit DataFrame.drop_duplicates if needed to ensure the back-link to here (unless other PR)

Reply:
let's keep that for another PR. But good idea to also add DataFrame.drop_duplicates and Series.duplicated

Reply:
adjusted in c300ea6